modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/security/permission/resource/AccountEntryModelResourcePermission.java

```diff
@@ -91,7 +100,9 @@ public boolean contains(
 		throws PortalException {
 
 		if (_accountEntryModelResourcePermission.contains(
-				permissionChecker, accountEntryId, actionId)) {
+				permissionChecker, accountEntryId, actionId) ||
```

Comment:
Supplier should not access all accounts, only the one he is member of.
Is this needed for showing order's details?

Reply:
Hi @luca-pellizzon , yes this is needed. The alternative is to change this method permission check: https://github.com/liferay-commerce/liferay-portal/blob/42246a7dccec49db7a18d3601dd299dd4b083974/modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemServiceImpl.java#L294

Reply:
Used in: https://github.com/liferay-commerce/liferay-portal/blob/42246a7dccec49db7a18d3601dd299dd4b083974/modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/frontend/data/set/provider/CommerceShippableOrderItemsFDSDataProvider.java#L81

Reply:
I guess we should use the localservice in this fds, you should be able to access orderItems as long as you can access the related order. We will fix the call later. Also I don't get why we filter by account there

Reply:
I will take care of this with a `localService` call instead.

Reply:
Address issues fixed here: https://github.com/liferay-commerce/liferay-portal/pull/3912/commits/3f7a41c86f21e3bf50e28cffb7cf1b08b58a0fd6